medusa/storage/google_storage.py

```diff
@@ -158,7 +158,7 @@ async def _download_blob(self, src: str, dest: str):
             stream = await self.gcs_storage.download_stream(
                 bucket=self.bucket_name,
                 object_name=object_key,
-                timeout=self.read_timeout if self.read_timeout is not None else -1,
+                timeout=self.read_timeout,
```

Comment:
**Question:** Disabling the timeout seemed to be done by setting the value to `-1` previously and now it's set to `None`. Does that work as well?

Reply:
I actually do not know.
Since it's really tedious to find this out empirically, let's just default to -1 for GCS. The commit https://github.com/thelastpickle/cassandra-medusa/pull/813/commits/3dd0a55431e402436bb693e75748122e9313ddb9 does that.